tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/ValidateManifestConsoleTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
You have now 2 x the same test class only with a different name. Remove this one.

Reply:
PLease comment or apply my requested change.

Reply:
It's still here. Di you forgot to push?

Reply:
Looks like you neeed to merge this file with the `AppManifestValidateConsoleTest`

You have now two test classes that test the same process.

Keep `AppManifestValidateConsoleTest` and copy the needed method `testValidateManifestReturnsSuccessCodeWhenValidationIsSuccessful` from `ValidateManifestConsoleTest` to `AppManifestValidateConsoleTest `.

Reply:
deleted ValidateManifestConsoleTest file and moved method into AppManifestValidateConsoleTest.